tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -409,10 +409,13 @@ public void SearchHandler(string query)
 
             if (system is ComputeSystemViewModel computeSystemViewModel)
             {
-                var providerName = computeSystemViewModel.ProviderDisplayName;
-                if (providerName != _selectedProvider)
+                if (_selectedProvider.ToString().Length > 0)

```

Comment:
Not blocking but since `_selectedProvider` is already a string we could do 

```C#
if (!string.IsNullOrEmpty(_selectedProvider))
{
    ....
}
```

or 

```C#
if (_selectedProvider.Length > 0)
{
    ....
}
```

Reply:
One question I do have though is whether this equality is also needed on line 402 in this file as well.

Reply:
Not needed in 402